### DISCUSSION TITLE
Getting error reactRender is not a function

### DISCUSSION THREAD

**Discussion by AileenSeditio** [ant-design/ant-design#52056] [Q&A] [answered]
I am getting an error  reactRender is not a function for the latest NextJs

here is my dependencies

`
  "dependencies": {
    "antd": "^5.22.5",
    "next": "15.1.0",
    "react": "^19.0.0",
    "react-dom": "^19.0.0",
    "react-router-dom": "^6.28.0"
  },`
  
  an example of using it is here
  
  `
"use client"

import React, { useState } from "react";
import { Radio } from 'antd';

const LazyTable = ({ className }) => {
  const [position, setPosition] = useState('end');
  
  return (
    <div className={`relative ${className}`}>
      <Radio.Group value={position} onChange={(e) => setPosition(e.target.value)}>
        <Radio.Button value="start">start</Radio.Button>
        <Radio.Button value="end">end</Radio.Button>
      </Radio.Group>
    </div>
  );
};

export default LazyTable;

`

error also happens on other buttons, drawers, and etc.

Here is a screenshot 

<img width="1023" alt="Screenshot 2024-12-19 at 4 00 14 PM" src="https://github.com/user-attachments/assets/f74785fc-7d7d-4589-af44-ceb631625600" />

**Comment by Wxh16144** [comment, marked as answer]
v5 not support React19

**Reply by Wxh16144** [reply]
View more info with https://github.com/ant-design/ant-design/discussions/51919
